.gitignore

```diff
@@ -63,3 +63,4 @@ plugin-transpiler/dist
 *-esbuild-meta.json
 *-esbuild-bundle-visualization.html
 .dlt
+*.db
```

Comment:
unrelated? unnecessary?

Reply:
Kept appearing in my diff whenever I pulled. But yeah unrelated